app/src/main/java/com/example/interesting_facts_about_numbers/ui/presentation/start_screen/StartScreenViewModel.kt

```diff
@@ -0,0 +1,82 @@
+package com.example.interesting_facts_about_numbers.ui.presentation.start_screen
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import androidx.navigation.NavHostController
+import com.example.interesting_facts_about_numbers.functionality.GetHistoryUseCase
+import com.example.interesting_facts_about_numbers.functionality.entity.NumberFact
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.*
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class StartScreenViewModel @Inject constructor(
+    private val getHistoryUseCase: GetHistoryUseCase,
+) : ViewModel() {
+
+    private val mutableStartScreenState: MutableStateFlow<String> =
```

Comment:
As I remember it supposed to be a temporary name 😄

Reply:
it was rename